### PR TITLE
docs(taiko-client): update readme how to do integration test

### PR DESCRIPTION
**Pull request by YoGhurt111**

### CODE REVIEW

File: packages/taiko-client/integration_test/README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # How to debug test cases?
+- set up config
+```
+export L2_NODE=l2_geth
```
Comment (davidtaikocha) [inline packages/taiko-client/integration_test/README.md:4]: By default which value will be used? `l2_reth`?

Comment (YoGhurt111) [reply packages/taiko-client/integration_test/README.md:4]: This config is just for local debugging.
In CI/CD, they have different envs in different git workflows.

Comment (YoGhurt111) [reply packages/taiko-client/integration_test/README.md:4]: `l2_geth ` is the default choice in `make test`.

Comment (YoGhurt111) [reply packages/taiko-client/integration_test/README.md:4]: When we debug locally, we won't use `entrypoint.sh`, so there is no default value for our local debugging.

Comment (davidtaikocha) [reply packages/taiko-client/integration_test/README.md:4]: Then let's set a default value in `entrypoint.sh`?

Comment (davidtaikocha) [reply packages/taiko-client/integration_test/README.md:4]: And then we can avoid setting this value manually here

Comment (YoGhurt111) [reply packages/taiko-client/integration_test/README.md:4]: We did not use `entrypoint.sh` in local debugging, so setting a default value in `entrypoint.sh` is not helpful.
We set up the test environment manually and just run the test case we want to debug.

**Review by davidtaikocha — APPROVED** [review]

**Review by dantaik — APPROVED** [review]

**Pull request merged**